app/src/test/java/com/codesoom/assignment/application/CatToyServiceTest.java

```diff
@@ -0,0 +1,227 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.ProductNotFoundException;
+import com.codesoom.assignment.domain.CatToyRepository;
+import com.codesoom.assignment.domain.Product;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+@DisplayName("CatToyService 클래스")
+class CatToyServiceTest {
+    private CatToyService catToyService;
+    private CatToyRepository catToyRepository;
+    static final Long EXIST_ID = 1L;
+    static final Long NOT_EXIST_ID = 999L;
+    Product product;
```

Comment:
```suggestion
    static final Long EXIST_ID = 1L;
    static final Long NOT_EXIST_ID = 999L;
    
    private CatToyService catToyService;
    private CatToyRepository catToyRepository;
    Product product;
```

상수와 변수를 분리해서 선언하면 좋을 것 같아요